server/broadcast.go

```diff
@@ -722,6 +722,10 @@ func selectOrchestrator(ctx context.Context, n *core.LivepeerNode, params *core.
 			bcastOS = bcastOS.OS().NewSession(pfx)
 		}
 
+		oinfo := n.OrchestratorPool.GetInfo(tinfo.Transcoder)
```

Comment:
Is this change related to your PR? This seems like a candidate for a separate PR.

Reply:
Yes, that what was making this bug hard to track down. `GetInfo` silently returns uninitialized structure in case there's no URL match. Ideally, we need to refactor it to return error, but it will require changes to the interface, probably should be a separate PR.